manta-crypto/src/ecc.rs

```diff
@@ -137,6 +137,9 @@ pub trait ScalarMul<COM = ()> {
 /// Elliptic Curve Pre-processed Scalar Multiplication Operation
 pub trait PreprocessedScalarMul<COM, const N: usize>: ScalarMul<COM> + Sized {
```

Comment:
I will add some tests for this in a day or two. Do not merge until this comment is marked resolved

Reply:
test added